it/uk/gov/hmrc/fileupload/VirusScanFileUploadISpec.scala

```diff
@@ -71,34 +70,14 @@ class VirusScanFileUploadISpec extends FileActions with EnvelopeActions with Eve
       eventually {
         Wiremock.scanFileCommandTriggered()
       }
-      (stubbedClamAVClient.send (_:Array[Byte])(_: ExecutionContext)).verify(*, *).twice()
-      (stubbedClamAVClient.checkForVirus () (_:ExecutionContext)).verify(*).twice()
-    }
-
-    "retry virus scanning if COMMAND READ TIMED OUT occurs until virus detected" in {
```

Comment:
Why was this test removed?

Reply:
there was two tests for this since the previous avclient took two methods - where now it takes one - so it only needs one test..

Reply:
Looks like I can preserve it